### PR TITLE
change price on storybook

### LINKED ISSUE #451: [FRONTEND] Update the pricing banner on homepage to say $60 for single friday/saturday
**Is your feature request related to a problem? Please describe.**

This thing:

Image

**BEFORE MERGING**

- [ ] Storybooks created where possible
- [ ] Tested with [nock](https://tanstack.com/query/v3/docs/framework/react/guides/testing) if data fetching
- [ ] Tests written for critical interactions
- [ ] PR Reviewed (For non-trivial changes)
- [ ] Changes tested after rebasing on master or merging in master (_hint_: `git fetch origin master:master`, then `git rebase master` or `git merge master`)
- [ ] All required PR checks passing

### PR DESCRIPTION
**Pull request by zlrkw11**
closes #451

### CODE REVIEW

**Comment by zlrkw11** [comment]
image
- desktop view port

**Comment by zlrkw11** [comment]
image
- mobile view port

**Comment by bcho892** [comment]
https://github.com/UoaWDCC/uasc-web/issues/451#issuecomment-2179982515

File: client/src/components/generic/PricingBanner/PricingBanner.story.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ type Story = StoryObj<typeof meta>
 export const Default: Story = {
   args: {
     headline: "Great nightly rates",
-    priceInformation: "$40 per night*",
+    priceInformation: "$60 per night*",
```
Comment (bcho892) [inline client/src/components/generic/PricingBanner/PricingBanner.story.tsx:15]: It's still wrong here

**Review by bcho892 — CHANGES_REQUESTED** [review]

**Review by bcho892 — APPROVED** [review]

File: client/src/pages/Home/sections/utils/Pricing.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ export const pricingNote =
 export const pricingBannerContent: PricingBannerContent = {
   headline: "Great nightly rates",
   priceInformation: "$40 per night*",
-  disclaimer: "*$50 when booking a single saturday"
+  disclaimer: "*$60 when booking a single saturday"
```
Comment (bcho892) [inline client/src/pages/Home/sections/utils/Pricing.ts:9]: mb, should say friday or saturday

**Pull request merged**